integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/CarbonHiveCommands.scala

```diff
@@ -41,3 +43,36 @@ case class CarbonDropDatabaseCommand(command: DropDatabaseCommand)
   }
 }
 
+case class CarbonSetCommand(command: SetCommand)
+  extends RunnableCommand {
+
+  override val output = command.output
+
+  override def run(sparkSession: SparkSession): Seq[Row] = {
+    if (command.kv.isDefined) {
```

Comment:
When only set , set key are used, which keys are displayed by spark, carbon also should be  consistant. like if spark shows default parameters carbon should also show. if spark shows property file content, carbon behaviour also should match